### ISSUE TITLE
BeepSaber hand tracking isn't quite right

### ISSUE THREAD

**Issue by Wavesonics** [NeoSpark314/godot_oculus_quest_toolkit#41]
The right hand saber's orientation is incorrect making it unplayable. Shouldn't be a hard fix I don't think?

**Comment by NeoSpark314** [comment]
Yes; I think this should be relatively easy to fix; but due to hand tracking lag I'm not sure it will be playable with hand tracking. 

Recently there were several improvements made by @Joanguitar that made beepsaper work better with OpenVR and also added playlist support; these improvements are currently in the https://github.com/NeoSpark314/godot_oculus_quest_toolkit/tree/beepsaber branch as I still need to test them with the quest; There was also a change for the orientation of the vive wands; we should look there and see if it can be generalized to oculus touch, wands and hands somehow;

I have some transform nodes set in the toolkit that are intended to solve these problems: https://github.com/NeoSpark314/godot_oculus_quest_toolkit/blob/b2f493f503d5e119f282fb461314454d8c1e3415/OQ_Toolkit/OQ_ARVRController/scripts/OQ_ARVRController.gd#L99 
maybe one of them already provides the correct transform for both hands and touch. I will check this.
